lib/phoenix_live_view/test/live_view_test.ex

```diff
@@ -470,7 +470,7 @@ defmodule Phoenix.LiveViewTest do
 
     assigns =
       Map.new(assigns)
-      |> Map.put_new(:__changed__, %{})
+      #|> Map.put_new(:__changed__, %{})
```

Comment:
Is `:__changed__` needed somewhere?
I commented it out because it causes these tests to fail:
-[live_component_test.exs:439](https://github.com/phoenixframework/phoenix_live_view/blob/7c153bdd87a7d356920ca264ea3a8aaeb899e684/test/phoenix_live_view/integrations/live_components_test.exs#L439)
-[live_component_test.exs:448](https://github.com/phoenixframework/phoenix_live_view/blob/7c153bdd87a7d356920ca264ea3a8aaeb899e684/test/phoenix_live_view/integrations/live_components_test.exs#L448)
When [this function](https://github.com/phoenixframework/phoenix_live_view/blob/7c153bdd87a7d356920ca264ea3a8aaeb899e684/test/phoenix_live_view/integrations/live_components_test.exs#L391) is called, it sends the `:__changed__ `assign to assign/2.

Reply:
I think it is fine to remove `__changed__` here. We use it only for change tracking and there is no change tracking happening in `render_component`.